apps/ledger-live-desktop/src/renderer/screens/manager/DeviceStorage/DeviceLanguage.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useCallback } from "react";
+import { Button, Flex, Icons } from "@ledgerhq/react-ui";
+import Text from "~/renderer/components/Text";
+import DeviceLanguageInstallation from "./DeviceLanguageInstallation";
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
+import { idsToLanguage, Language, DeviceInfo } from "@ledgerhq/types-live";
+import { useTranslation } from "react-i18next";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+
+type Props = {
+  // this makes sure that this component is only rendered if languageId is present in deviceInfo
+  deviceInfo: DeviceInfo & { languageId: number };
+  device: Device;
+};
+
+const DeviceLanguage: React.FC<Props> = ({ deviceInfo, device }: Props) => {
+  const [isLanguageInstallationOpen, setIsLanguageInstallation] = useState(false);
+  const [deviceLanguage, setDeviceLanguage] = useState<Language>(
+    idsToLanguage[deviceInfo.languageId],
+  );
+  const [selectedLanguage, setSelectedLanguage] = useState<Language>(deviceLanguage);
+
+  const refreshDeviceLanguage = useCallback(() => {
+    setDeviceLanguage(selectedLanguage);
+  }, [setDeviceLanguage, selectedLanguage]);
+
+  const { t } = useTranslation();
+
+  return (
+    <Flex alignItems="center">
+      <Icons.LanguageMedium color="neutral.c80" size={24} />
+      <Flex ml={1} mr={5}>
+        <Text ff="Inter|SemiBold" color="palette.text.shade40" fontSize={4}>
+          {t("deviceLocalization.language")}
+        </Text>
+      </Flex>
+      <Button
```

Comment:
I think it might look better here with the `Link` component as there is no border & no background in the button but still the borders are highlighted when hovering which looks a bit weird.
Example here https://github.com/LedgerHQ/ledger-live/pull/1352/files#diff-7ebcc21939fa15f1a1fec90643baa559d5ea2921a0f6493f04a29bf25bbdbb8e

Reply:
Done